src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -672,7 +694,7 @@ static void define_media_references(py::module m) {
 
 
     py::class_<MissingReference, MediaReference,
-               managing_ptr<MissingReference>>(m, "MissingReference", py::dynamic_attr())
+               managing_ptr<MissingReference>>(m, "MissingReference", py::dynamic_attr(), "Represents media for which a concrete reference is missing.")
```

Comment:
Note that a MissingReference may have useful metadata, even if the location of the media is not known.